File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 7/IAcceptHangout.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_7
{
	internal class IAcceptHangout : StoryChoice
	{
		internal IAcceptHangout() : base(StoryChoice.GetLocalizedString("ch7IAcceptHangout"), 7022)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch7HungOutWithHewitt = HewittHangout.Accepted;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_7/IAcceptHangout.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_7;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.HewittHangout;

/**
 * Created by christopherraleigh on 2016-05-19.
 */
public class IAcceptHangout extends StoryChoice {
    public IAcceptHangout() {
        super(R.string.ch7IAcceptHangout, 7022);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh7HungOutWithHewitt(HewittHangout.Accepted);
        super.select();
    }
}
